Resources/Prototypes/Entities/Buildings/instruments.yml

```
@@ -55,6 +55,21 @@
      sprite: Objects/Instruments/otherinstruments.rsi
      state: minimoog
 
+ - type: entity
+   name: church organ
+   parent: BaseInstrument
+   id: ChurchorganInstrument
```

This needs to follow the [YAML naming guidlines](https://hackmd.io/@ss14/code-quality-control#YAML)
```suggestion
   id: ChurchOrganInstrument
```